tests/TestFormula.elm

```diff
@@ -182,9 +183,11 @@ isSubformulaOfTests =
         , describe "Conj" <| binIsSubformulaTests Conj a b
         , describe "Disj" <| binIsSubformulaTests Conj a b
         , describe "Impl" <| binIsSubformulaTests Conj a b
+        , describe "Equiv" <| binIsSubformulaTests Conj a b
```

Comment:
Fix these to really test the respective connective, e.g, `binIsSubformulaTests Impl a b`.
Fix `binIsSubformulaTests` to really use the connective from the first argument instead of `Conj`.